indy_node/utils/node_control_tool.py

```diff
@@ -109,10 +107,8 @@ def _update_package_cache(cls):
             timeout=TIMEOUT)
 
         if ret.returncode != 0:
-            msg = 'Upgrade failed: _get_deps_list returned {}'.format(
-                ret.returncode)
-            logger.error(msg)
-            raise Exception(msg)
+            raise Exception('_get_deps_list returned {}'
```

Comment:
Can we have more descriptive exception here, like "Can not update packages, since 'apt update' returned {}"?